### DISCUSSION TITLE
lastfm scrobbler

### DISCUSSION THREAD

**Discussion by atmix** [AzuraCast/AzuraCast#3683] [Ideas]
Hello everyone. Please tell me if it is possible to set up scrobbling for lastfm.

**Comment by Vaalyn** [comment]
Hey, would you mind describing `scrobbler` and the kind of integration you have in mind for this?

I haven't used last.fm so I'm unfamiliar with what their scrobbler / scrobbling is and what can be done with it.

**Reply by atmix** [reply]
Liquidsoap has such a function. In Azuracast, it just remains to add the liquidsoap-plugin-lastfm plugin.
https://www.liquidsoap.info/doc-dev/reference-extras.html#lastfm.submit
https://github.com/savonet/liquidsoap/blob/master/libs/lastfm.liq

**Reply by BusterNeece** [reply]
This should probably be a request over on our [FeatureUpvote page](https://features.azuracast.com/) so we can consider it and our community can vote on it along with other requests.
